src/build.sh

```diff
@@ -70,11 +70,11 @@ function kernel_build()
   fi
 
   if [[ -n "$cross_compile" ]]; then
-    cross_compile="CROSS_COMPILE=$cross_compile"
+    cross_compile=" CROSS_COMPILE=$cross_compile "
   fi
 
   if [[ -n "$menu_config" ]]; then
-    command="make -j ARCH=$arch $cross_compile $menu_config"
+    command="make -j ARCH=$arch$cross_compile$menu_config"
```

Comment:
Since you are concatenating variables, let's use: `${arch}${cross_compile}${menu_config}`.
https://kworkflow.org/content/codingstyle.html#string-concatenation
(same for this commit)